agents/src/pipeline/agent_output.ts

```diff
@@ -0,0 +1,185 @@
+// SPDX-FileCopyrightText: 2024 LiveKit, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+import type { AudioFrame } from '@livekit/rtc-node';
+import { log } from '../log.js';
+import { SynthesizeStream, type TTS } from '../tts/index.js';
+import { AsyncIterableQueue, CancellablePromise, Future, gracefullyCancel } from '../utils.js';
+import type { AgentPlayout, PlayoutHandle } from './agent_playout.js';
+
+export type SpeechSource = AsyncIterable<string> | string | Promise<string>;
+
+export class SynthesisHandle {
```

Comment:
Maybe I'm misreading the code. But it seems like classes like `SynthesisHandle`, `SpeechHandle`, `DeferredValidation` are exposed to the user. Let's keep them private if this is the case

Reply:
ah right, exported them for the other files and exported them here by accident